nginx/multi-channel.yaml

```diff
@@ -1,2 +1,10 @@
 .+:
-  channel: default
+  - baseHref: /en
```

Comment:
please don't change anything here. This is supposed to be the working default for the multi channel.